api/v1beta1/types.go

```diff
@@ -579,6 +579,37 @@ type SubnetSpec struct {
 	SubnetClassSpec `json:",inline"`
 }
 
+// Network Interfaces to attach to each VM
+// +optional
+type AzureNetworkInterface struct {
+	// The subnet to place the interface in
```

Comment:
```suggestion
	// The subnet to place the interface in.
```
Could we end the comments with periods? Not sure if the linters will complain but it's more consistent in any case.